sammba/io_conversions/bruker_dicom.py

```diff
@@ -12,13 +12,10 @@
 import nibabel
 from .utils import _rotate_affine
 
-DICOM_EXTENSIONS = [".dcm", ".ima", ".dicom"]
-
-
 def _is_dicom(filename):

```

Comment:
may be you can call the function `_is_bruker_dicom`, what do you think ?

Reply:
for the forseeable future all our dicom operations will be on Bruker dicoms so am not against the idea of naming all our dicom functions after bruker (or maybe more specifically, paravision)

Reply:
like `_is paravision_dicom` ?

Reply:
yes

Reply:
though for the purposes of SEO calling it bruker dicom might be better. plus it's two syllables and four letters fewer

Reply:
I would go for `_is_paravision_dicom` because this is an internal non exposed function. For code readability, this hints that we are only dealing with paravision format. If anyone wants to access this function, he will have to import it from `io_conversions.bruker_dicom` so he will understand we are dealing with bruker dicoms.
BTW, are there other popular tools apart from paravision used by small animal brukers ?

Reply:
that's what I don't know. the company makes all sorts of other instrumentation, but I don't know what the names are of the software packages used to run those other products, and we don't have any DICOMs from them (assuming the other packages export to DICOM too, which they probably do/should)

Reply:
btw I just realised there is a direct way to test if it is a Bruker/Paravision DICOM....the manufacturer name and software (including version I think) are both tags in the DICOM

Reply:
The relevant dcmdump outputs are:

(0008,0070) LO [Bruker BioSpin MRI GmbH]                #  24, 1 Manufacturer
(0018,1020) LO [ParaVision Acquisition 6.0.1\ParaVision 6.0.1] #  46, 2 SoftwareVersions

The second is probably the most useful. We have only ever tested ParaVision 6* DICOMs so maybe have a warning if this flag is not there.

Reply:
(0002,0002) UI =EnhancedMRImageStorage                  #  28, 1 MediaStorageSOPClassUID
(0008,0016) UI =EnhancedMRImageStorage                  #  28, 1 SOPClassUID

might be useful too if someone tries to cheat and rename a non-conformant DICOM to force it through